src/lib/learn-client/api/bookmark/delete-bookmark.ts

```diff
@@ -0,0 +1,40 @@
+import { ApiTutorial } from 'lib/learn-client/api/api-types'
+import { destroy, toError } from 'lib/learn-client'
+import { BOOKMARK_API_ROUTE } from '.'
+import { SessionData } from 'types/auth'
+
+interface DeleteBookmarkOptions {
+	accessToken: SessionData['accessToken']
+	tutorialId: ApiTutorial['id']
+}
+
+type DeleteBookmarkResult = void
+
+/**
+ * Deletes the bookmark for the given `tutorialId`. If successful, returns
+ * nothing. Otherwise throws an error.
+ *
+ * https://digital-api-specs.vercel.app/learn#tag/Bookmarks/paths/~1bookmarks~1{tutorial_id}/delete
+ */
+const deleteBookmark = async ({
+	accessToken,
+	tutorialId,
+}: DeleteBookmarkOptions): Promise<DeleteBookmarkResult> => {
+	// Build the API route to make a request to
+	const requestRoute = `${BOOKMARK_API_ROUTE}/${tutorialId}`
+
+	// Make the GET request
```

Comment:
```suggestion
	// Make the DELETE request
```

Reply:
Fixed in https://github.com/hashicorp/dev-portal/pull/828/commits/5d4fea55d6f0bd69b0f17672f3d0ae8938010452!